src/withStyles.jsx

```diff
@@ -197,13 +199,13 @@ export function withStyles(
     WithStyles.displayName = `withStyles(${wrappedComponentName})`;
     WithStyles.contextTypes = contextTypes;
     if (WrappedComponent.propTypes) {
-      WithStyles.propTypes = deepmerge({}, WrappedComponent.propTypes);
+      WithStyles.propTypes = objectAssign({}, WrappedComponent.propTypes);
       delete WithStyles.propTypes[stylesPropName];
       delete WithStyles.propTypes[themePropName];
       delete WithStyles.propTypes[cssPropName];
     }
     if (WrappedComponent.defaultProps) {
-      WithStyles.defaultProps = deepmerge({}, WrappedComponent.defaultProps);
+      WithStyles.defaultProps = objectAssign({}, WrappedComponent.defaultProps);
```

Comment:
this can/should use object spread, with these babel settings: https://github.com/airbnb/rheostat/blob/master/.babelrc#L4